packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchModalOverlay.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <div v-if="isBelowMid">
+        <full-address-overlay-search
+            is-full-height
+            is-scrollable
+            :has-close-button="false"
+            :is-open="shouldDisplayModalOverlay"
+            :class="$style['c-megaModal--isFullWidth']"
+            @close="onModalClose">
+            <div
+                :class="[
+                    $style['c-fullAddressFinder-inner'],
+                    { [$style['has-elevation']]: hasInputElevation }
+                ]">
+                <label
+                    :class="{
+                        [$style['c-search-label']]: true,
+                        [$style['c-search-label--noBorder']]: true
+                    }">
+                    <input
+                        :aria-label="copy.fieldLabel"
+                        :class="{
+                            [$style['c-search-input']]: true,
+                            [$style['c-search-input-overlay']]: true,
+                            [$style['is-notEmpty']]: true
+                        }"
+                        type="search"
+                        @input="address = $event.target.value">
+                    <span :class="$style['c-search-placeholder']">{{ copy.fieldLabel }}</span>
+                </label>
+
+                <cancel-button
+                    type="button"
+                    :class="$style['c-fullAddressFinder-btn']"
+                    @click.native="onModalClose">
+                    {{ copy.fullAddressSearchSuggestions.buttonCancel }}
```

Comment:
If you update f-button to the latest version you won't need `type="button"` and `.native` on a click event

Reply:
Fixed.